src/core/react-query/auth/mutations.ts

```diff
@@ -18,3 +18,24 @@ export const useLoginMutation = () =>
       }));
     },
   });
+
+export const useCreateApiToken = () =>
+  useMutation<string, unknown, string>({
```

Comment:
```suggestion
  useMutation<string>({
```

No need to specify it twice if it is not being transformed.

Reply:
got this error if it not define like that
![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/11846133/f8b92149-aff6-449e-8d4c-633a65ff53bd)
![image](https://github.com/ShokoAnime/Shoko-WebUI/assets/11846133/e00ec4ca-7b39-42c2-a1ef-276e52d0d3fb)

Reply:
Ohh. I was thinking of the query type syntax. The 2nd `string` here is the input param type for the mutation function